docs/conf.py

```diff
@@ -46,6 +47,4 @@
 # -- Options for extlinks ----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html#configuration
 
-extlinks = {
-    "pypi": ("https://pypi.org/project/%s", "%s"),
```

Comment:
IIUC, the `:pypi:` role is also (randomly) provided by sphinx-issues? If we're no longer using extlinks at all, shall we remove it from the extensions list to make that clear?